doc/user/olm-catalog/generating-a-csv.md

```diff
@@ -17,23 +17,34 @@ Operator SDK projects have an expected [project layout][doc-project-layout]. In
 
 `gen-csv` reads these files and adds their data to a CSV in an alternate form.
 
-By default, a `deploy/olm-catalog/csv-config.yaml` file is generated when `gen-csv` is first run. The defaults written in the following fields contain paths to the aforementioned files. From the [design doc][doc-csv-design]:
+The following example config containing default values should be copied and written to `deploy/olm-catalog/csv-config.yaml`:
+
+```yaml
+crd-cr-paths:
+- deploy/crds
+operator-path: deploy/operator.yaml
+role-paths:
+- deploy/role.yaml
+```
+
+Explanation of all config fields:
+
+- `crd-cr-paths`: list of strings - a list of CRD and CR manifest file/directory paths. Defaults to `[deploy/crds]`.
```

Comment:
Just noticed this but what do the square brackets [ ] signify?
Line 33 below doesn't have them: `Defaults to deploy/operator.yaml`

Reply:
Ah never mind, it's a list.

Reply:
A list/slice.